### DISCUSSION TITLE
Thank You Again + README + Question

### DISCUSSION THREAD

**Discussion by FelipeRearden** [jglev/obsidian-carry-forward#2] [General]
Hello @jglev !!!!!

I was just gonna say thank you for creating this amazing plugin but then I saw that this is another plugin from you (the creator of the spectacular Apply Patterns) I was very very very happy!!!!!!!!!!1

This plugin is amazing and you again help me to make my workflow easier in Obsidian. 
- I had 2 QuickAdd macros to transfer the text and another to transfer the blockref (link) of only one line. It was a pain to do this 100 times a day; 
- Now one hotkey solves all of my problems. 

Your approach to create the links, alias and the regex feature is fantastic.

Everything you do is a game changer for me THANK YOU AGAIN !!!! 

### README 

Just to let you know that the links in the readme are related to the Apply Patterns plugin :)

### Question ?

I'd like to know if I can request FR for this plugin too . 

Thanks again and have a great day!

**Comment by jglev** [comment]
Thank you for your kind words; they mean a lot to me!

> Just to let you know that the links in the readme are related to the Apply Patterns plugin :)

Thank you! I've addressed this just now, in `v1.3.0`.

> I'd like to know if I can request FR for this plugin too .

Yes, of course! : )

**Reply by FelipeRearden** [reply]
> Thank you for your kind words; they mean a lot to me!

I'm glad :). Be sure that my words came from my ❤️ . 

I love your work. 
- Carry-Forward is my time life saver. I can work on Obsidian faster in Obsidian because I have this jewel everyday with me 

- Apply-Patterns is my superhero: every time I face a text transformation I know Apply will help me

- Paste to Current indentation is new to me (the new version). I am testing a workflow that I think it will save me a lot of time (I will talk to you about that in a few days)

> Thank you! I've addressed this just now, in `v1.3.0`.

😄 

> Yes, of course! : ) 

Thank very much ! I have one more big FR for Carry that I want to bring to you. It's very simple to explain but I want to take some time to write to you my real workflow for this FR to make this 100% crystal clear 

Have a great day!!!!!!!
